### PR TITLE
Add access to multimedia directories

### LINKED ISSUE #86: Use YunoHost's multimedia feature
### Describe the bug

Seriously, that's a multimedia app, let's let it use the multimedia directories.

### PR DESCRIPTION
**Pull request by tituspijean**
Closes #86 

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
[Test Badge](https://ci.pijean.ovh/ci/job/353)
[](https://ci.pijean.ovh/ci/job/353)

File: scripts/install [hunk 1]
```diff
@@ -178,6 +178,16 @@ ynh_setup_source --dest_dir="/var/lib/jellyfin/plugins/LDAP Authentication" --so
...
 ynh_add_config --template="LDAP-Auth.xml" --destination="/var/lib/jellyfin/plugins/configurations/LDAP-Auth.xml"
 
+#=================================================
+# YUNOHOST MULTIMEDIA INTEGRATION
+#=================================================
+ynh_script_progression --message="Adding multimedia directories..." --weight=2
+
+# Build YunoHost multimedia directories
+ynh_multimedia_build_main_dir
+# Allow nextcloud to write into these directories
```
Comment (Salamandar) [inline scripts/install:188]: Jellyfin*

File: scripts/restore [hunk 2]
```diff
@@ -184,6 +184,16 @@ ynh_script_progression --message="Starting a systemd service..." --weight=1
...
 ynh_systemd_action --service_name=$app --action="restart" --log_path="systemd" --line_match="Started Jellyfin Media Server" --timeout=15
 
+#=================================================
+# YUNOHOST MULTIMEDIA INTEGRATION
+#=================================================
+ynh_script_progression --message="Adding multimedia directories..." --weight=2
+
+# Build YunoHost multimedia directories
+ynh_multimedia_build_main_dir
+# Allow nextcloud to write into these directories
```
Comment (Salamandar) [inline scripts/restore:194]: Same here

File: scripts/install (hunk 1 above)
Comment (tituspijean) [reply scripts/install:188]: Aaah busted. I shamelessly copied Nextcloud's code. 😅

**Pull request merged**